js/kronolith.js

```diff
@@ -3853,8 +3853,17 @@ KronolithCore = {
             data.owner_input = null;
         }
         if (data.type == 'holiday') {
-            this.insertCalendarInList('holiday', data.driver, Kronolith.conf.calendars.holiday[data.driver]);
-            this.toggleCalendar('holiday', data.driver);
+			// checking if a holiday is already in the list
+            let check = this.getCalendarList(data.type).textContent.replace(/[^\w\s]/gi, '');
+            if (check === data.driver){
```

Comment:
Does this work if you already have multiple holiday calendars?

Reply:
thanks! will fix that